core/src/main/java/tc/oc/pgm/listeners/PGMListener.java

```diff
@@ -258,6 +260,26 @@ public void handleItemPickup(final PlayerPickupItemEvent event) {
     if (nearestPlayer != event.getPlayer()) event.setCancelled(true);
   }
 
+  @EventHandler
+  public void lockFireTick(final MatchLoadEvent event) {
+    event.getMatch().getWorld().setGameRuleValue(DO_FIRE_TICK, Boolean.toString(false));
+  }
+
+  // unlock firespread if not disabled by <gamerule>
+  @EventHandler
+  public void unlockFireTick(final MatchStartEvent event) {
+    ImmutableMap<String, String> lockFireTick =
```

Comment:
These are gamerules, NOT lockFireTick, give proper names to variables

Reply:
I forgot to simplify this.